examples/config/config

```diff
@@ -321,6 +329,7 @@ set history_disable_easter_egg 1
 # Set function shortcut
 @cbind  s<var:>_<value:>_  = set %1 %2
 # Exit binding
+@cbind  q                  = exit
```

Comment:
I've many times closed my WM by accident by accidentally hitting the wrong key. Quiting something should by default be a keycombo, not  a single key IMO.